modules/history/src/views/full/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import style from './style.scss'
+import moment from 'moment'
+
+import { MessagesViewer } from './message'
+import { ConversationPicker } from './conversation'
+
+export default class FullView extends React.Component {
```

Comment:
Not very important, but the constructor isn't necessary here. You can initialize the state without `this` like
state = {}, and move the date logic in the componentDidMount instead